.github/workflows/colima-tests.yml

```diff
@@ -102,14 +111,17 @@ jobs:
         uses: actions/cache/restore@v3
         if: ${{ github.ref != 'refs/heads/master' }}
         with:
-          path: ~/Library/Caches/Homebrew
+          path: ${{ steps.homebrew-cache.outputs.dir }}
           key: ${{ runner.os }}-homebrew-cache-${{ steps.get-date.outputs.date }}
           restore-keys: |
             ${{ runner.os }}-homebrew-cache-
 
-      - name: Set up Homebrew
-        id: set-up-homebrew
-        uses: Homebrew/actions/setup-homebrew@master
+      - name: Clean up Homebrew
+        run: |
+          brew update
```

Comment:
`brew update` is done automatically by `brew upgrade` these days, but this does no harm.

Reply:
Did not find any hint on that in the docs. Only cleanup is mentioned. https://docs.brew.sh/Manpage#upgrade-options-outdated_formulaoutdated_cask-